src/Annotator.js

```diff
@@ -542,13 +546,9 @@ class Annotator extends EventEmitter {
 
         const pageThreads = this.threads[pageNum] || {};
         Object.keys(pageThreads).forEach((threadID) => {
-            const thread = pageThreads[threadID];
```

Comment:
Annotations with disabled types should be filtered out by this point so this extra check is unecessary